tests/integration/release/test_release.py

```diff
@@ -35,17 +41,117 @@ def test_column_noising(
 
 
 def test_row_noising_omit_row_or_do_not_respond(
-    noised_data: pd.DataFrame,
     dataset_name: str,
     config: dict[str, Any],
 ) -> None:
-    """Tests that omit_row and do_not_respond row noising are being applied"""
-    idx_cols = IDX_COLS.get(dataset_name)
-    original = get_unnoised_data(dataset_name)
-    original_data = original.data.set_index(idx_cols)
-    noised_data = noised_data.set_index(idx_cols)
+    """Tests that the correct noising is applied to each dataset when
+    noising with omit_row and do_not_respond by checking the config."""
+    noise_config: NoiseConfiguration = get_configuration(config)
+    noise_types = [
+        noise_type
+        for noise_type in [NOISE_TYPES.omit_row.name, NOISE_TYPES.do_not_respond.name]
+        if noise_config.has_noise_type(dataset_name, noise_type)
+    ]
+
+    if dataset_name in [
+        DATASET_SCHEMAS.census.name,
+        DATASET_SCHEMAS.acs.name,
+        DATASET_SCHEMAS.cps.name,
+    ]:
+        # Census and household surveys have do_not_respond and omit_row.
+        # For all other datasets they are mutually exclusive
+        with check:
+            assert len(noise_types) == 2
+    else:
+        with check:
+            assert len(noise_types) < 2
+
+
+@pytest.mark.parametrize("expected_noise", ["default", 0.01])
+def test_omit_row(
+    expected_noise: str | float,
+    unnoised_dataset: Dataset,
+    dataset_name: str,
+    fuzzy_checker: FuzzyChecker,
+) -> None:
+    """Tests that omit_row noising is being applied at the expected proportion."""
+    original_data = unnoised_dataset.data
+    config_dict = get_single_noise_type_config(dataset_name, NOISE_TYPES.omit_row.name)
+
+    if expected_noise != "default":
+        config_dict[dataset_name][Keys.ROW_NOISE][NOISE_TYPES.omit_row.name][
+            Keys.ROW_PROBABILITY
+        ] = expected_noise
+        config = NoiseConfiguration(LayeredConfigTree(config_dict))
+    else:
+        config = NoiseConfiguration(LayeredConfigTree(config_dict))
+        expected_noise: float = config.get_row_probability(
+            dataset_name, NOISE_TYPES.omit_row.name
+        )
+    dataset_schema = DATASET_SCHEMAS.get_dataset_schema(dataset_name)
+    dataset = Dataset(dataset_schema, original_data, 0)
```

Comment:
Let's move all test seeds into a SEED constant defined at the top (or maybe in a conftest? )